man/crystal.1

```diff
@@ -0,0 +1,259 @@
+.\"Crystal Programming Language
+.Dd February 11, 2016
+.Dt CRYSTAL(1) "" "Crystal Programmers Reference Guide"
```

Comment:
I guess this was inspired by `ruby`'s man page title. A somewhat misleading title, in this case.